### DISCUSSION TITLE
ossec/wazuh - disable or suppress unwanted alert

### DISCUSSION THREAD

**Discussion by ArronJablonowski** [Security-Onion-Solutions/securityonion#2573] [Unsupported Versions] [answered]
Security Onion 2.3.21 on Ubuntu 18.04 - 
How can I suppress or disable an unwanted ossec (wazuh) alert? I'm being flooded with "Apparmor DENIED" notifications in SOC. Which I believe to be sid 52002.

**Comment by ArronJablonowski** [comment, marked as answer]
I was able to resolve the issue and suppress the alerting by greping for the rule.uuid (52002) in /opt/so/rules/hids/ruleset/rules/ to find the rule set that contained rule.uuid 52002,  './rules/0315-apparmor_rules.xml '  I then changed the rule's Level from level="3" to level="0". Finally I restarted the Wazuh service (sudo so-wazuh-restart) and the alert has since stopped.

```
grep -r 52002 /opt/so/rules/hids/ruleset/rules/ 
sudo nano /opt/so/rules/hids/ruleset/rules/rules/0315-apparmor_rules.xml
Change value to: 
   <rule id="52002" level="0">
       Save changes to 0315-apparmor_rules.xml. 
sudo so-wazuh-restart     
```

**Reply by trb143** [reply]
This will fail when Wazuh updates and over rights its files.  See https://documentation.wazuh.com/4.0/learning-wazuh/replace-stock-rule.html for the correct and recommended way to amend rule priorities.
